src/plugins/janus_streaming.c

```diff
@@ -4226,6 +4239,14 @@ static json_t *janus_streaming_process_synchronous_request(janus_streaming_sessi
 			char *old_metadata = mp->metadata;
 			char *new_metadata = g_strdup(json_string_value(md));
 			mp->metadata = new_metadata;
+			if (send_edited_event == TRUE) {
+				if (old_metadata == NULL && new_metadata == NULL) {
+					send_edited_event = FALSE;
+				}
+				if (old_metadata != NULL && new_metadata != NULL && !strcmp(old_metadata, new_metadata)) {
+					send_edited_event = FALSE;
+				}
```

Comment:
Code style nit: both checks above have a simple one-line action, so we usually don't put curly brackets in that case. It looks like you can combine the two ifs in a single one with an OR, though, e.g.:

	if ((old_metadata == NULL && new_metadata == NULL) ||
			(old_metadata != NULL && new_metadata != NULL && !strcmp(old_metadata, new_metadata)))
		send_edited_event = FALSE;

Reply:
I have combined all the 3 ifs... I'm unsure about coding style here... Any URL for coding style?

Reply:
Fair point :smile:  We don't have a specific set of rules, even if @atoppi tried to codify them in #2910. In general it's not a big deal for us anyway, you'll see several pieces of code that don't comply for instance: I just happen to point out those that I notice in order to try and provide some consistency.